### PR TITLE
TLS Everywhere configuration for standalone

### PR DESCRIPTION
**Pull request by xek**

### CODE REVIEW

**Comment by xek** [comment]
recheck

**Comment by xek** [comment]
recheck

**Comment by vakwetu** [comment]
recheck

**Comment by vakwetu** [comment]
recheck

**Comment by vakwetu** [comment]
recheck

**Comment by vakwetu** [comment]
recheck

**Comment by vakwetu** [comment]
recheck

**Comment by vakwetu** [comment]
recheck

**Comment by cschwede** [comment]
recheck

**Comment by vakwetu** [comment]
recheck

File: devsetup/scripts/standalone.sh [hunk 1]
```diff
@@ -96,8 +99,8 @@ sudo dnf install -y podman python3-tripleoclient util-linux lvm2 cephadm
 # STEP_CONFIG in container-puppet-* containers.
 sudo dnf install -y https://kojihub.stream.centos.org/kojifiles/packages/podman/4.6.0/1.el9/x86_64/podman-4.6.0-1.el9.x86_64.rpm
 
-sudo hostnamectl set-hostname standalone.localdomain
-sudo hostnamectl set-hostname standalone.localdomain --transient
+sudo hostnamectl set-hostname standalone.ooo.test
+sudo hostnamectl set-hostname standalone.ooo.test --transient
```
Comment (fao89) [inline devsetup/scripts/standalone.sh:103]: probably needs: https://github.com/openstack-k8s-operators/data-plane-adoption/pull/395

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
the "EDPM part" completed, but it is failing on Nova FFU now

**Comment by gibizer** [comment]
```
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service [None req-04ee4faf-cf27-41b1-b6f0-90df44f4385b - - - - - -] Error starting thread.: nova.exception.InvalidConfiguration: My node 32b9ce55-446d-43d2-a92c-5fd3a816f63c has host 'standalone.localdomain' but my host is 'standalone.ooo.test'; Possible rename detected, refusing to start!
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service Traceback (most recent call last):
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service   File "/usr/lib/python3.9/site-packages/oslo_service/service.py", line 806, in run_service
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service     service.start()
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service   File "/usr/lib/python3.9/site-packages/nova/service.py", line 162, in start
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service     self.manager.init_host(self.service_ref)
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service   File "/usr/lib/python3.9/site-packages/nova/compute/manager.py", line 1619, in init_host
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service     self._check_for_host_rename(nodes_by_uuid)
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service   File "/usr/lib/python3.9/site-packages/nova/compute/manager.py", line 1564, in _check_for_host_rename
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service     raise exception.InvalidConfiguration(
Apr  3 17:44:00 standalone nova_compute[353327]: 2024-04-03 17:44:00.879 2 ERROR oslo_service.service nova.exception.InvalidConfiguration: My node 32b9ce55-446d-43d2-a92c-5fd3a816f63c has host 'standalone.localdomain' but my host is 'standalone.ooo.test'; Possible rename detected, refusing to start!
```

So somehow the compute node got renamed in the porcess

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by xek** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by xek** [comment]
I removed the depends-on, since the other patch is not required for CI

**Comment by xek** [comment]
recheck

**Comment by xek** [comment]
recheck

**Review by fao89 — APPROVED** [review]

**Comment by marios** [comment]
@xek can we talk about this in today's data plane adoption call please 

i want to partially revert this, at least the bits under 'tripleo' directory (not standalone).

it blocks our testing and ongoing work on https://issues.redhat.com/browse/OSPRH-6205 

cc @cescgina

**Pull request merged**